### PR TITLE
Add commands for backup management

### LINKED ISSUE #27: Add command for downloading backups

### PR DESCRIPTION
**Pull request by martin-helmich**
Fixes #27

- Clean up backup commands
- Implement backup download
- Add "resume" functionality
- Update README

### CODE REVIEW

**Comment by LukasFritzeDev** [comment]
Ist es dann sinnvoll auch gleich ein `backup create` zu machen?

File: src/commands/project/backup/download.tsx [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+          if (r.data.export?.phase === "Completed") {
+            return r.data.export;
+          }
+
+          return null;
+        }, 3600);
+      },
+    );
+
+    if (!exp.downloadURL) {
```
Comment (LukasFritzeDev) [inline src/commands/project/backup/download.tsx:140]: wtf is `exp`? experience? 😜

File: src/rendering/react/components/ProjectBackup/ProjectBackupDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+export const ProjectBackupDetails: FC<{
+  projectBackup: BackupProjectBackup;
+}> = ({ projectBackup }) => {
+  const schedule = projectBackup.parentId
+    ? useProjectBackupSchedule(projectBackup.parentId)
+    : undefined;
+
+  return (
+    <SingleResult
```
Comment (LukasFritzeDev) [inline src/rendering/react/components/ProjectBackup/ProjectBackupDetails.tsx:19]: should the project name be displayed in the details?

File: src/commands/project/backup/download.tsx (hunk 1 above)
Comment (martin-helmich) [reply src/commands/project/backup/download.tsx:140]: What is `export`? A reserved keyword. 😜

File: src/rendering/react/components/ProjectBackup/ProjectBackupDetails.tsx (hunk 2 above)
Comment (martin-helmich) [reply src/rendering/react/components/ProjectBackup/ProjectBackupDetails.tsx:22]: Good idea. Fixed in 2dfb1b2

File: src/commands/project/backup/download.tsx (hunk 1 above)
Comment (martin-helmich) [reply src/commands/project/backup/download.tsx]: Renamed to `backupExport` in b776db2

**Review by LukasFritzeDev — APPROVED** [review]

**Pull request merged**